src/Json.Schema.Validation.UnitTests/ValidatorTests.cs

```diff
@@ -1064,6 +1064,82 @@ public void Serialize(IXunitSerializationInfo info)
   }
 }",
                 MakeErrorMessage(2, 8, "a", ErrorNumber.DependentPropertyMissing, "x", "\"y\"", "\"y\"")
+                ),
+            new TestCase(
+                "String: format: valid with date-time format constraints",
+                @"{
+                  ""type"": ""string"",
+                  ""format"": ""date-time""
+                }",
+                "\"2023-02-03T12:34:56.789Z\""
+                ),
+            new TestCase(
+                "String: format: invalid date-time format string",
+                @"{
+                  ""type"": ""string"",
+                  ""format"": ""date-time""
+                }",
+                "\"failure-string\"",
+                MakeErrorMessage(1, 16, string.Empty, ErrorNumber.StringDoesNotMatchFormat, FormatAttributes.DateTime, "failure-string")
+                ),
+            new TestCase(
+                "String: format: valid with uri format constraint",
+                @"{
+                  ""type"": ""string"",
+                  ""format"": ""uri""
+                }",
+                "\"https://www.example.com\""
```

Comment:
>"\"https://www.example.com\"" [](http://example.com/codeflow?start=16&length=29)

could you add a file uri test case as well, thanks

Reply:
Added test cases for file and mailto uri as well